handlers.go

```diff
@@ -475,7 +475,13 @@ func forwardEvent(falcopayload types.FalcoPayload) {
 	}
 
 	if config.OTLP.Traces.Endpoint != "" && (falcopayload.Priority >= types.Priority(config.OTLP.Traces.MinimumPriority)) && (falcopayload.Source == syscall || falcopayload.Source == syscalls) {
-		go otlpClient.OTLPTracesPost(falcopayload)
+		go otlpTracesClient.OTLPTracesPost(falcopayload)
+	}
+
+	if config.OTLP.Metrics.Endpoint != "" &&
+		(falcopayload.Priority) >= types.Priority(config.OTLP.Metrics.MinimumPriority) &&
+		(falcopayload.Source == syscall || falcopayload.Source == syscalls) {
```

Comment:
no need to check for the source for the metrics, it's only relevant for the traces